lib/ansible/modules/cloud/azure/azure_rm_keyvault.py

```diff
@@ -0,0 +1,430 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Ian Philpot, <ian.philpot@microsoft.com>;
```

Comment:
We have a newer license header which is shorter, see https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/azure/azure_rm_virtualmachine.py for more info.